security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,70 @@ These release notes track the development of {cert-manager-operator}.
+* You can now configure cloud credentials for the {cert-manager-operator} on GCP clusters with Workload Identity and without Workload Identity.
+
+[id="cert-manager-dns01-acme-aws-1.11.1"]
+==== Configuring an ACME issuer to solve DNS01 challenges by using ambient credentials on AWS
+* You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using ambient credentials on AWS.
+
+[id="cert-manager-dns01-acme-gcp-1.11.1"]
+==== Configuring an ACME issuer to solve DNS01 challenges on GCP
+* You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using ambient and explicit credentials on GCP.
+
+[id="cert-manager-dns01-acme-azure-1.11.1"]
+==== Configuring an ACME issuer to solve DNS01 challenges on Microsoft Azure
+* You can use the {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using explicit credentials on Microsoft Azure.
+
+[id="cert-manager-operator-1.11.1-bug-fixes"]
+=== Bug fixes
+
+* Previously, the `cm-acme-http-solver` pod did not use the latest published Red Hat image `registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9`. With this release, the `cm-acme-http-solver` pod uses the latest published Red Hat image `registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9`. (link:https://issues.redhat.com/browse/OCPBUGS-10821[*OCPBUGS-10821*])
+
+* Previously, the {cert-manager-operator} did not support changing labels for cert-manager pods such as controller, CA injector, and Webhook pods. With this release, you can add labels to cert-manager pods. (link:https://issues.redhat.com/browse/OCPBUGS-8466[*OCPBUGS-8466*])
+
+* Previously, the `operatorLogLevel` field did not update the log verbosity level in the {cert-manager-operator}. Now, the `operatorLogLevel` field properly updates the log verbosity level in the {cert-manager-operator}. (link:https://issues.redhat.com/browse/OCPBUGS-9994[*OCPBUGS-9994*])
+
+* Previously, when uninstalling the {cert-manager-operator}, if you select the *Delete all operand instances for this operator* checkbox in the {product-title} web console, the Operator was not uninstalled properly. Now, the {cert-manager-operator} is now properly uninstalled. (link:https://issues.redhat.com/browse/OCPBUGS-9960[*OCPBUGS-9960*])
+
+* Previously, the {cert-manager-operator} did not support using Google workload identity federation. Now, the {cert-manager-operator} supports using Google workload identity federation. (link:https://issues.redhat.com/browse/OCPBUGS-9998[*OCPBUGS-9998*])
+
+* Previously, if you set the log level and the log level verbosity for the CA injector and Webhook components, the {cert-manager-operator} used to panic and fail to update the CA injector and Webhook pods. Now, the {cert-manager-operator} updates the CA injector and Webhook pods with the log level and log level verbosity.  (link:https://issues.redhat.com/browse/OCPBUGS-14129[*OCPBUGS-14129*])
```

Comment:
Need not mention [OCPBUGS-14129](https://issues.redhat.com/browse/OCPBUGS-14129). Because "previously" released 1.10.2 does not have this bug. It is only introduced during 1.11.1.